os_pkrvars/ubuntu/ubuntu-20.04-aarch64.pkrvars.hcl

```diff
@@ -1,10 +1,12 @@
-os_name                 = "ubuntu"
-os_version              = "20.04"
-os_arch                 = "aarch64"
-iso_url                 = "http://cdimage.ubuntu.com/releases/20.04/release/ubuntu-20.04.5-live-server-arm64.iso"
-iso_checksum            = "sha256:e42d6373dd39173094af5c26cbf2497770426f42049f8b9ea3e60ce35bebdedf"
-hyperv_generation       = 2
-parallels_guest_os_type = "ubuntu"
-vbox_guest_os_type      = "Ubuntu_64"
-vmware_guest_os_type    = "ubuntu-64"
-boot_command            = ["<wait><esc>linux /casper/vmlinuz quiet autoinstall ds='nocloud-net;s=http://{{.HTTPIP}}:{{.HTTPPort}}/ubuntu/'<enter>initrd /casper/initrd<enter>boot<enter>"]
+os_name                   = "ubuntu"
+os_version                = "20.04"
+os_arch                   = "aarch64"
+iso_url                   = "http://cdimage.ubuntu.com/releases/20.04/release/ubuntu-20.04.5-live-server-arm64.iso"
+iso_checksum              = "sha256:e42d6373dd39173094af5c26cbf2497770426f42049f8b9ea3e60ce35bebdedf"
+hyperv_generation         = 2
+parallels_guest_os_type   = "ubuntu"
+vbox_guest_os_type        = "Ubuntu_64"
+vmware_guest_os_type      = "arm-ubuntu-64"
+boot_command              = ["<wait><esc>linux /casper/vmlinuz quiet autoinstall ds='nocloud-net;s=http://{{.HTTPIP}}:{{.HTTPPort}}/ubuntu/'<enter>initrd /casper/initrd<enter>boot<enter>"]
+vmware_cdrom_adapter_type = "sata"
+vmware_disk_adapter_type  = "sata"
```

Comment:
I'm thinking this makes sense and is the most compatible. Can you try making it the default for all vmware builds? If we set it at the variables level and then remove local variable in the pkr-sources file that it'd work for all builds and then we wouldn't need these extra variable overrides for all aarch64 builds.

Reply:
Agreed!  I was trying to minimize change scope but if you want repo-wide that’s definitely better/cleaner on all fronts.  My visibility is just a bit limited to make that call.

Reply:
worth testing